scripts/biomcli

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
```

Comment:
i think this needs to be named `biom` otherwise it'll break command line backwards compatibility